Jenkinsfile

```diff
@@ -1419,8 +1418,7 @@ pipeline {
                             filename 'Dockerfile.centos.7'
                             dir 'utils/docker'
                             label 'docker_runner'
-                            additionalBuildArgs "-t ${sanitized_JOB_NAME}-centos7 " +
-                                '$BUILDARGS_QB_TRUE' +
+                            additionalBuildArgs dockerBuildArgs(qb: quickbuild()) +
```

Comment:
Ditto.

It's probably worth mentioning that it's probable that this `-t ...` argument could be added to [dockerBuildArgs()](https://github.com/daos-stack/pipeline-lib/blob/master/vars/dockerBuildArgs.groovy), but that's a separate effort than what you are fixing here.